pkg/scheduler/framework/plugins/serviceaffinity/service_affinity_test.go

```diff
@@ -618,7 +619,7 @@ func TestPreFilterDisabled(t *testing.T) {
 	}
 	cycleState := framework.NewCycleState()
 	gotStatus := p.Filter(context.Background(), cycleState, pod, nodeInfo)
-	wantStatus := framework.AsStatus(fmt.Errorf(`error reading "PreFilterServiceAffinity" from cycleState: not found`))
+	wantStatus := framework.AsStatus(fmt.Errorf(`error reading "PreFilterServiceAffinity" from cycleState: %w`, errors.New("not found")))
```

Comment:
Instead, export the error in `framework/cycle_state.go`? That is the benefit of wrapping, that we can compare the exact error. Now, we shouldn't be using reflect, but that's a separate problem.